src/coreclr/pal/tests/palsuite/runpaltests.sh

```diff
@@ -79,8 +72,6 @@ then
   fi
 fi
 
-cd $PAL_TEST_OUTPUT_DIR
```

Comment:
There are few other `cd xxx` and `pushd xxx`/`popd` in this script, do we need those?

Reply:
I remove all what I can. And nothing else matters.

Reply:
Now the log output is much cleaner. :+1:

Reply:
What is important for me as newcomer, is that I should not learn which "errors" are not actual errors. I always trying to assume that infra is right, and I'm wrong. I understand that path which I take to find this issue not that straight :smile:

Reply:
There is always room for improvement. :)
PAL tests are mainly used (and become important) when support for a new platform (OS, architecture, libc) is added. Since that does not happen quite often, so this infra gets stale from time to time. These tests were added in the CI pipeline (namely, `innerloop`), so it is at least functional.